app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/HelpCenterGraph.kt

```diff
@@ -62,5 +79,18 @@ fun NavGraphBuilder.helpCenterGraph(
         },
       )
     }
+    composable<HelpCenterDestinations.CommonClaim> {
+      CommonClaimDestination(
+        commonClaim = commonClaim,
+        navigateUp = navigator::navigateUp,
+        navigateBack = navigator::popBackStack,
+      )
+    }
+    composable<HelpCenterDestinations.Emergency> {
+      EmergencyDestination(
+        emergencyData = emergency,
+        navigateUp = navigator::navigateUp,
+      )
+    }
```

Comment:
Are these some help center specific versions of those two screens, or is it that those two screens are now kinda thought of as being part of the help center feature?

Reply:
My thought is that these are part of the help center now. We moved them from the "other services" bottom sheet to the help center quick actions.